Assets/Tests/InputSystem/Plugins/LinuxTests.cs

```diff
@@ -226,6 +227,7 @@ private struct TestSDLJoystick : IInputStateTypeInfo
 
         public TestSDLJoystick WithButton(SDLButtonUsage usage, bool value = true)
         {
+            Debug.Assert((int)SDLButtonUsage.Count <= 32);
```

Comment:
Best to put a message on these. No clue what's up with our stack traces but you get real bad traces for `Debug.Asserts`. Never seems to point to the right place in the code. Putting a message on these helps finding the assert that got triggered.

Reply:
fixed